tools/experiments/experiments.js

```diff
@@ -303,6 +303,18 @@ const EXPERIMENTS = [
     spec: 'TODO',
     cleanupIssue: 'TODO',
   },
+  {
```

Comment:
`amp-orientation-observer` was just removed intentionally in https://github.com/ampproject/amphtml/pull/17156. We shouldn't be re-adding it here. =)